src/LiveSplit.DetailedTimer/UI/Components/DetailedTimerSettings.Designer.cs

```diff
@@ -1094,20 +1187,22 @@ private void InitializeComponent()
             "Current Timing Method",
             "Real Time",
             "Game Time"});
-            this.cmbTimingMethod.Location = new System.Drawing.Point(156, 33);
+            this.cmbTimingMethod.Location = new System.Drawing.Point(208, 42);
+            this.cmbTimingMethod.Margin = new System.Windows.Forms.Padding(4);
             this.cmbTimingMethod.Name = "cmbTimingMethod";
-            this.cmbTimingMethod.Size = new System.Drawing.Size(286, 21);
+            this.cmbTimingMethod.Size = new System.Drawing.Size(382, 24);
             this.cmbTimingMethod.TabIndex = 3;
             this.cmbTimingMethod.SelectedIndexChanged += new System.EventHandler(this.cmbTimingMethod_SelectedIndexChanged);
             // 
             // DetailedTimerSettings
             // 
-            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(8F, 16F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
             this.Controls.Add(this.tableLayoutPanel1);
+            this.Margin = new System.Windows.Forms.Padding(4);
             this.Name = "DetailedTimerSettings";
-            this.Padding = new System.Windows.Forms.Padding(7);
-            this.Size = new System.Drawing.Size(459, 991);
+            this.Padding = new System.Windows.Forms.Padding(9);
+            this.Size = new System.Drawing.Size(612, 1220);
```

Comment:
Did you forget to change the scaling back to 100% when working on this component?